lib/components/BraidTestProvider/BraidTestProvider.docs.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { seekAnz } from '../../themes';
+import { ComponentDocs } from '../../../site/src/types';
+import { Stack, Text } from '..';
+
+const docs: ComponentDocs = {
+  category: 'Logic',
+  description: (
+    <Stack space="large">
+      <Text>Alternative to `BraidProvider` for unit test environments.</Text>
```

Comment:
Could we add a strong warning to not import this in your actual App? Just to be sure.

Reply:
Maybe even throw an error?

Reply:
How would we detect that?

Reply:
`NODE_ENV` of development or production?

Reply:
Worried that some teams will have that set for some reason in there test setup.

Reply:
Yeah, fair point. I'll add it to the docs.